runtime/compiler/optimizer/InlinerTempForJ9.cpp

```diff
@@ -1231,18 +1248,49 @@ TR_J9InlinerPolicy::createUnsafePutWithOffset(TR::ResolvedMethodSymbol *calleeSy
       indirectAccessTreeTop->getNode()->setIsUnsafeStaticWrtBar(true);
       }
 
-   TR_OpaqueClassBlock *javaLangClass = comp()->getClassClassPointer(/* isVettedForAOT = */ true);
-   // If we are not able to get javaLangClass it is still inefficient to put direct Access far
-   // So in that case we will generate lowTagCmpTest to branch to indirect access if true
-   bool needNotLowTagged = javaLangClass != NULL  || conversionNeeded ;
-   TR::TreeTop *lowTagCmpTree = genClassCheckForUnsafeGetPut(offset, needNotLowTagged);
-
    if (tracer()->debugLevel())
       {
       debugTrace(tracer(), "\t After genClassCheckForUnsafeGetPut, lowTagCmpTree dump:\n");
       comp()->getDebug()->print(comp()->getOutFile(), lowTagCmpTree);
       }
 
+#if defined(TR_TARGET_64BIT)
```

Comment:
For testing this is fine, but you'll need to replace that with `J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION`